api/CSSStyleDeclaration.json

```diff
@@ -0,0 +1,484 @@
+{
+  "api": {
+    "CSSStyleDeclaration": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSStyleDeclaration",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "1"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "1"
+          },
+          "firefox_android": {
+            "version_added": "4"
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": true
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      }
+    },
+    "cssText": {
```

Comment:
This isn't a sub-element of CSSStyleDeclaration (neither are any of the other subfeatures), the indentation needs to be fixed  

The file with code folding:

<img width="443" alt="screen shot 2018-06-09 at 3 40 37 pm" src="https://user-images.githubusercontent.com/2977353/41196365-7a2ffe60-6bfb-11e8-881c-75355aca4730.png">

Reply:
After a quick check I _think_ this is the only PR you did this on.

Reply:
Thanks and sorry, fixed now in 33b2a3d.